sdks/java/core/src/main/java/org/apache/beam/sdk/io/ReadableFileCoder.java

```diff
@@ -20,31 +20,63 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
-import org.apache.beam.sdk.coders.AtomicCoder;
-import org.apache.beam.sdk.coders.Coder;
-import org.apache.beam.sdk.coders.VarIntCoder;
+import java.util.Collections;
+import java.util.List;
+import org.apache.beam.sdk.coders.*;
```

Comment:
this asterisk-based import is causing the PreCommit to fail (we try to avoid this sort of import) - can you import only the classes you need?